ci/helm/environments/functional-tests.yaml

```diff
@@ -1,11 +1,16 @@
-# enabled conditions
+# these values are required by helmfile-base.yaml.gotmpl releases
 nuxeo:
   enabled: true
 mongodb:
   enabled: true
 postgresql:
   enabled: false
+opensearch:
+  enabled: false
+kafka:
+  enabled: false
```

Comment:
Should we enable them by the way, to be closer from the production stack?

Reply:
Yes we can, just wanted to make iso deployments for better review. We can go one ahead.